AdminWebsite/AdminWebsite/ClientApp/src/app/shared/menus/case-types-menu/case-types-menu.component.html

```diff
@@ -8,17 +8,17 @@
     bindLabel="item"
     bindValue="item"
     formControlName="selectedCaseTypes"
-    [placeholder]="'Select case types'"
+    [placeholder]="'Select Services'"
```

Comment:
sorry to nitpick, but probably leave the s lowercase: `Select services`

Reply:
done

Reply:
done